resources/views/themes/default1/layouts/front/master.blade.php

```diff
@@ -543,9 +542,9 @@
       var email = $('#newsletterEmail').val();
       $('#mailchimp-subscription').html("<i class='fa fa-circle-o-notch fa-spin fa-1x fa-fw'></i>Please Wait...");
       $.ajax({
-        type: 'GET',
+        type: 'POST',
         url: '{{url("mail-chimp/subcribe")}}',
```

Comment:
Does this route not processed by VerfifyCSRF middleware?

Reply:
Fixed